fdbserver/Ratekeeper.actor.cpp

```diff
@@ -520,13 +520,17 @@ class RatekeeperImpl {
 
 						bool returningTagsToProxy{ false };
 						if (SERVER_KNOBS->ENFORCE_TAG_THROTTLING_ON_PROXIES) {
-							reply.proxyThrottledTags = self.tagThrottler->getProxyRates(self.grvProxyInfo.size());
-							returningTagsToProxy =
-							    reply.proxyThrottledTags.present() && reply.proxyThrottledTags.get().size() > 0;
+							auto proxyThrottledTags = self.tagThrottler->getProxyRates(self.grvProxyInfo.size());
```

Comment:
Does this call or the client one have any side effects? If not, you could eliminate this entire block with the knob starting with the `if` at 522.

Reply:
It has the side effect of tracing the computed rates, which is what we still want in report-only mode.